### PR TITLE
Create devmeeting2024.md

### PR DESCRIPTION
**Pull request by tjeerdijk**
news item for the 2024 OpenMS developer's conference

### CODE REVIEW

File: content/en/news/devmeeting2024.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+title: OpenMS developers conference 10-16 March 2024, near Malaga, Spain
+authors: ["Tjeerd Dijkstra, Timo Sachsenberg"]
+date: 2024-01-19
+summary: We are hosting a developers conference near Malaga, Spain! Click the header above for more information.
+---
+
+The OpenMS developers conference brings together people working in computational mass spectrometry to shape the future development of OpenMS. It is targeted to core developers, new developers, and potential future contributors. Short talks, developer tutorials, and code sprints will be intertwined. Participants will have the opportunity to design custom tools and workflows together with instructors. New developers will receive help getting started in OpenMS development. Progress will be tracked on our GitHub page with a resume of each day. For more details ping Tjeerd Dijkstra or Timo Sachsenberg on Gitter.
```
Comment (timosachsenberg) [inline content/en/news/devmeeting2024.md:8]:
```suggestion
The OpenMS developers conference brings together people working in computational mass spectrometry to shape the future development of OpenMS. It is targeted to core developers, new developers, and potential future contributors. Short talks, developer tutorials, and code sprints will be intertwined. Participants will have the opportunity to design custom tools and workflows together with instructors. New developers will receive help getting started in OpenMS development. Progress will be tracked on our GitHub page with a resume of each day. For more details ping Tjeerd Dijkstra or Timo Sachsenberg on discord or email.
```

**Comment by timosachsenberg** [comment]
hmm any idea why this fails? can you retrigger it

**Comment by jpfeuffer** [comment]
Long standing issue. Seems to be caching.

**Comment by timosachsenberg** [comment]
So just merge?

**Comment by jpfeuffer** [comment]
No, indeed retrigger without cache preferably.

**Comment by timosachsenberg** [comment]
Retriggered … worked on netlify with deleted cache.

**Comment by timosachsenberg** [comment]
IMG_6412

**Comment by timosachsenberg** [comment]
Something else that needs to be done?

**Comment by jpfeuffer** [comment]
Only the NEtlify comment will be updated I think. Maybe the integration is broken.

Does anyone still need the file that raises the error?? applications.md.bup?
I left it there so people can update their tools to the new structure but since it does not seem to be a popular task I think we can also just get rid of it.

**Comment by jpfeuffer** [comment]
Someone would need to figure out how caching at netlify works.

**Comment by timosachsenberg** [comment]
What is this file used for? First time I hear of it.

I just saw this comment in the original bulk PR.
```
Note: People have to update their summaries for their application pages. See epifany.md. <--! more --> or any other way of writing summaries:
https://gohugo.io/content-management/summaries/
```
Ok to get the new page up (it is kind of urgent). Can I just deploy it through netlify? Does anyone know?

**Pull request merged**